language: php
name: PHP
results:
- exit_code: 0
  program: "<?php\n// Add two numbers x and y\n// >>> add(2, 3)\n// 5\n// >>> add(5,\
    \ 7)\n// 12\nfunction add($x, $y) {\n    return $x + $y;\n}\n\nfunction candidate(...$args)\
    \ {\n    return add(...$args);\n}\n\nfunction test(): void {\n    if (candidate(0,\
    \ 1) !== 1) { throw new Exception(\"Test failed!\"); }\n    if (candidate(1, 0)\
    \ !== 1) { throw new Exception(\"Test failed!\"); }\n    if (candidate(2, 3) !==\
    \ 5) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5, 7) !== 12)\
    \ { throw new Exception(\"Test failed!\"); }\n    if (candidate(7, 5) !== 12)\
    \ { throw new Exception(\"Test failed!\"); }\n}\n\ntest();"
  status: OK
  stderr: ''
  stdout: ''
  timestamp: 1659990885
- exit_code: 255
  program: "<?php\n// Add two numbers x and y\n// >>> add(2, 3)\n// 5\n// >>> add(5,\
    \ 7)\n// 12\nfunction add($x, $y) {\n    return $x + y;\n}\n\nfunction candidate(...$args)\
    \ {\n    return add(...$args);\n}\n\nfunction test(): void {\n    if (candidate(0,\
    \ 1) !== 1) { throw new Exception(\"Test failed!\"); }\n    if (candidate(1, 0)\
    \ !== 1) { throw new Exception(\"Test failed!\"); }\n    if (candidate(2, 3) !==\
    \ 5) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5, 7) !== 12)\
    \ { throw new Exception(\"Test failed!\"); }\n    if (candidate(7, 5) !== 12)\
    \ { throw new Exception(\"Test failed!\"); }\n}\n\ntest();"
  status: Exception
  stderr: ''
  stdout: "\nFatal error: Uncaught Error: Undefined constant \"y\" in /tmp/tmpntbi1u1u.php:8\n\
    Stack trace:\n#0 /tmp/tmpntbi1u1u.php(12): add(0, 1)\n#1 /tmp/tmpntbi1u1u.php(16):\
    \ candidate(0, 1)\n#2 /tmp/tmpntbi1u1u.php(23): test()\n#3 {main}\n  thrown in\
    \ /tmp/tmpntbi1u1u.php on line 8\n"
  timestamp: 1659990885
